game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModel.java

```diff
@@ -35,46 +41,67 @@
     UUID
   }
 
-  private final Messengers messengers;
   private final boolean admin;
+  private final Timer gamePoller;
 
   // these must only be accessed in the swing event thread
-  private final List<Tuple<UUID, GameDescription>> gameList = new ArrayList<>();
-  private final ILobbyGameBroadcaster lobbyGameBroadcaster =
-      new ILobbyGameBroadcaster() {
+  private final List<Tuple<String, GameDescription>> gameList = new ArrayList<>();
+  private final LobbyGameUpdateListener lobbyGameBroadcaster =
+      new LobbyGameUpdateListener() {
         @Override
-        public void gameUpdated(final UUID gameId, final GameDescription description) {
-          assertSentFromServer();
-          updateGame(gameId, description);
+        public void gameUpdated(final String gameId, final LobbyGame lobbyGame) {
+          Optional.ofNullable(gameId)
+              .ifPresent(id -> updateGame(id, GameDescription.fromLobbyGame(lobbyGame)));
```

Comment:
`LobbyGameListing` is a `game id` and `lobbyGame`
The server uses game listing and lobby game. Those should be the types we are getting back from server and are sending to the server. GameDescription is still around almost as legacy at this point. It seemed risky in terms of accuracy of change and scope to remove GameDescription at this juncture.

Reply:
Side note, part of the motivation is try and remove usages of node. It looks to be really common that we pass a node object around just to get player name from it. This is only bad because we wind up sending and exposing  IP address information.

Reply:
Second side-note, lobbyGameListing is an nicer compared to GameDescription because it uses primitives. A rule of thumb I try to follow is to use primitives at system boundaries when coupling systems together. Once data traverses that boundary, encapsulate the information into domain objects (strongly typed objects) and use the strong typing internally. This means for example we are not dependent on the serialization of `Instant`, or that we have an exact `uuid`, or how that is represented as a JSON object, and/or how a `Node` is represented.

Please note that `LobbyGame` has only primitives.